integration-tests/integration/statements/submit-statement.spec.js

```diff
@@ -42,7 +42,7 @@ context('Submit statement', () => {
     }
 
     const submitStatementPage = SubmitStatementPage.verifyOnPage()
-    submitStatementPage.offenderName().contains('Norman Smith (A1234AC)')
+    submitStatementPage.offenderName().contains('Smith, Norman')
```

Comment:
Andy said that it should be first name last name, any place where it isn't a list